event_dispatcher.rst

```diff
@@ -322,6 +322,17 @@ its name:
 
     $ php bin/console debug:event-dispatcher kernel.exception
 
+.. versionadded:: 5.3
+
+    You can also find listeners to event names partially matching a string. Example
+    to show all events and their listeners related to ``kernel``, your app or Security:
+
+    .. code-block:: terminal
+
+        $ php bin/console debug:event-dispatcher kernel
+        $ php bin/console debug:event-dispatcher App
+        $ php bin/console debug:event-dispatcher Security
+
```

Comment:
```suggestion

to get everything matching "kernel" (i.e. ``kernel.exception``, ``kernel.response`` etc.) use:

.. code-block:: terminal

    $ php bin/console debug:event-dispatcher kernel

.. versionadded:: 5.3

    The ability to match partial event names was introduced in Symfony 5.3.
```

The advantage is, that we can just remove the versionadded in Symfony 6 docs, without rearranging the code examples itself.

Reply:
I see. Thank you

Reply:
Thank you, shall we add examples for App and or Security? Are these event names or does it matches a namespace?